tests/runtime/basic

```diff
@@ -72,3 +72,13 @@ NAME pid fails validation with non-numeric argument
 RUN bpftrace -p not_a_pid
 EXPECT ERROR: pid 'not_a_pid' is not a valid number.
 TIMEOUT 1
+
+NAME libraries under /usr/include are in the search path
+RUN bpftrace -e "$(echo "#include <sys/types.h>"; echo "BEGIN { exit(); }")" 2>&1
+EXPECT [^(file not found)]
```

Comment:
I don't think this regex is doing what you want. It'll just match a single character that isn't one of: `'(', 'f', 'i', 'l', 'e', ' ', 'n', 'o', 't', ' ', 'f', 'o', 'u', 'n', 'd', ')'`

Reply:
Good catch, thanks!